### PR TITLE
Fix: Drop support for PHP 5.3

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] drops support for PHP 5.3

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "mindscape/raygun4php",
     "type": "library",
-    "description": "Raygun.io client for PHP 5.3, to send errors and exceptions automatically.",
```
Comment (localheinz) [inline composer.json]: I think it's best to drop the PHP version from the description.

**Comment by robbieaverill** [comment]
How about we drop PHP 5 support entirely and release in a new major release line (2.x)? PHP 7.1 is already out of active support, we could either to PHP 7.0+ or drop those and go straight for PHP 7.2+

**Comment by localheinz** [comment]
@robbieaverill

Sounds good to me!

**Comment by localheinz** [comment]
Not sure how to interpret https://travis-ci.org/MindscapeHQ/raygun4php/jobs/500269304#L500-L505

```
Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - This package requires php ^5.4 but your HHVM version does not satisfy that requirement.
```

**Comment by robbieaverill** [comment]
Not sure, have restarted the build

File: composer.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "keywords": ["raygun", "errors", "exceptions", "logging"],
     "license": "MIT",
     "require": {
-        "php": ">=5.3.0"
+        "php": "^5.4"
```
Comment (robbieaverill) [inline composer.json]: I think this is what's causing the problem. Can you do `>=5.4.0` instead?

**Pull request merged**